From: SeongJae Park <sj@kernel.org>
Date: Fri, 20 Jun 2025 17:11:25 -0700
Subject: [PATCH] ==== damon-based interleave prototype ====

mm/damon: add DAMOS_MIGRATE_{HOT,COLD} destination nodes and weights

DAMOS_MIGRATE_{HOT,COLD} action can migrate pages to only single node.
It could be useful to allow it migrates pages to multiple nodes with
different weights.  A work for dynamic interleaving[1] is in progress.
We also discussed this could be useful for memory tiering, e.g., when we
want to move pages to multiple noes of same tier.

Extend the API and ABI interface for specifying the multiple destination
nodes and their weights.

Note that this RFC is a prototype of the interface change for Bijan's
interleaving work[1].  Hence this patch series is only implementing the
interface part, not the real implementation of the migration behavior.

[1] https://lore.kernel.org/20250620180458.5041-1-bijan311@gmail.com
[2] https://lore.kernel.org/20240408175228.91414-1-sj@kernel.org

Cc: Bijan Tabatabai <bijantabatab@micron.com>
Cc: Honggyu Kim <honggyu.kim@sk.com>
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_m66kbfql | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_m66kbfql

diff --git a/hkml_cv_bogus/hkml_cv_bogus_m66kbfql b/hkml_cv_bogus/hkml_cv_bogus_m66kbfql
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

